docs/DxcOnUnix.rst

```diff
@@ -0,0 +1,133 @@
+======================================
+DirectXShaderCompiler on Linux & macOS
+======================================
+
+.. contents::
+   :local:
+   :depth: 3
+
+Introduction
+============
+
+DirectXShaderCompiler (DXC) is based on LLVM/Clang, which is originally
+cross-platform. However, to support HLSL, certain Windows specific techniques
+(like COM, SAL, etc.) are introduced to solve technical issues on the Windows
+platform, which also makes DXC not compilable/runnable on non-Windows platforms.
+
+Upon `several <https://github.com/Microsoft/DirectXShaderCompiler/issues/1082>`_
+requests from the community, we have started the effort to enable compilation
```

Comment:
Nit: Oh, sorry. I meant that we can have two links here, one to Microsoft#1082, another to Microsoft#1236, behind "several" and "requests" to show that there are several requests.

Reply:
Ah. Done